### PR TITLE
CVSL-1740 patch dependencies

### PR DESCRIPTION
**Pull request by andrewrlee**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "record-build-info": "node ./bin/record-build-info",
     "lint": "eslint . --cache --max-warnings 0",
     "typecheck": "tsc",
-    "test": "jest --collectCoverage=true",
+    "test": "jest --maxWorkers=2 --collectCoverage=true",
```
Comment (andrewrlee) [inline package.json:24]: I thought this might help speed of tests on snooz-oma.

File: package.json [hunk 2]
```diff
@@ -112,7 +112,7 @@
     "connect-flash": "^0.1.1",
     "connect-redis": "^7.1.1",
     "csurf": "^1.11.0",
-    "date-fns": "^2.30.0",
+    "date-fns": "^3.4.0",
```
Comment (andrewrlee) [inline package.json:115]: this is a bit of a [scary jump](https://github.com/date-fns/date-fns/blob/main/CHANGELOG.md)!

File: package.json [hunk 3]
```diff
@@ -129,12 +129,12 @@
...
     "number-to-words": "^1.2.4",
     "nunjucks": "^3.2.4",
-    "passport": "^0.6.0",
+    "passport": "^0.7.0",
     "passport-oauth2": "^1.8.0",
     "qrcode": "^1.5.3",
     "redis": "^4.6.13",
     "reflect-metadata": "^0.1.14",
-    "sqs-consumer": "^7.5.0",
+    "sqs-consumer": "^9.1.0",
```
Comment (andrewrlee) [inline package.json:137]: bit of [a leap](https://github.com/bbc/sqs-consumer/releases) here too

File: package.json (hunk 2 above)
Comment (louisenorris) [reply package.json:115]: 😱

**Review by louisenorris — APPROVED** [review]
LGTM

**Pull request merged**